rules/aws/persistence_rds_group_creation.toml

```diff
@@ -0,0 +1,62 @@
+[metadata]
+creation_date = "2021/06/05"
+maturity = "production"
+updated_date = "2021/06/05"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Identifies when a RDS Security Group is Created.
+"""
+false_positives = [
+    """
+    A RDS group may be created by a system or network administrator. 
```

Comment:
```suggestion
    A RDS security group may be created by a system or network administrator. Verify whether the user identity, user agent, and/or hostname
    should be making changes in your environment. Security group creations from unfamiliar users or hosts should be
    investigated. If known behavior is causing false positives, it can be exempted from the rule.
```
re-lint